components/Collections/examples/CollectionsEditingExample.m

```diff
@@ -100,18 +103,36 @@ - (UICollectionReusableView *)collectionView:(UICollectionView *)collectionView
   // TODO: (shepj) Pending fix for
   // https://github.com/material-components/material-components-ios/issues/1208
   // we must call super which registers classes for supplemental header and footer in editing mode
-  UICollectionReusableView *supplementaryView = [super collectionView:collectionView
-                                    viewForSupplementaryElementOfKind:kind
-                                                          atIndexPath:indexPath];
-  if (supplementaryView) {
-    return supplementaryView;
+  //  UICollectionReusableView *supplementaryView = [super collectionView:collectionView
+  //                                    viewForSupplementaryElementOfKind:kind
+  //                                                          atIndexPath:indexPath];
+  //  UICollectionReusableView *supplementaryView =
+  //  [collectionView dequeueReusableSupplementaryViewOfKind:kind
+  //  withReuseIdentifier:HEADER_REUSE_IDENTIFIER forIndexPath:indexPath]; if (supplementaryView) {
+  //    return supplementaryView;
+  //  }
+
+  NSString *reuseIdentifier;
```

Comment:
This will alter the use of the collection. In the MDCCollectionViewController class, within this method we also call updateControllerWithInfoBar:(MDCCollectionInfoBar *)infoBar which in short, assigns the supp view as a property to the controller so it can update it with each selection/deselection. So it's not just about updating the identifiers.